static/monorail.yml

```diff
@@ -1432,7 +1437,7 @@ paths:
         - name: q
           in: query
           description: query object to pass through to waterline.
-          required: false
+          required: true
```

Comment:
@jlongever The tests and library are correct now.  We shouldn't be setting the required flag for an optional parameter even if it's not actually enforced by the API.